apps/web/src/components/MobileBottomMenuBar/MobileBottomMenuBar.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { createStyles, makeStyles, Theme } from '@material-ui/core/styles';
+
+import Button from '@material-ui/core/Button';
+import EndEventButton from '../Buttons/EndEventButton/EndEventButton';
+import LeaveEventButton from '../Buttons/LeaveEventButton/LeaveEventButton';
+import useRoomState from '../../hooks/useRoomState/useRoomState';
+import useVideoContext from '../../hooks/useVideoContext/useVideoContext';
+import { Typography, Grid } from '@material-ui/core';
+import ToggleAudioButton from '../Buttons/ToggleAudioButton/ToggleAudioButton';
+import ToggleChatButton from '../Buttons/ToggleChatButton/ToggleChatButton';
+import ToggleVideoButton from '../Buttons/ToggleVideoButton/ToggleVideoButton';
+import ToggleParticipantWindowButton from '../Buttons/ToggleParticipantWindow/ToggleParticipantWindowButton';
+import { useAppState } from '../../state';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    container: {
+      backgroundColor: theme.palette.background.default,
+      bottom: 0,
+      left: 0,
+      right: 0,
+      height: `${theme.footerHeight}px`,
+      position: 'fixed',
+      display: 'flex',
+      padding: '0 1.43em',
+      zIndex: 10,
+      [theme.breakpoints.down('sm')]: {
+        height: `${theme.mobileFooterHeight}px`,
+        padding: 0,
+      },
+    },
+    screenShareBanner: {
+      position: 'fixed',
+      zIndex: 8,
+      bottom: `${theme.footerHeight}px`,
+      left: 0,
+      right: 0,
+      height: '104px',
+      background: 'rgba(0, 0, 0, 0.5)',
+      '& h6': {
+        color: 'white',
+      },
+      '& button': {
+        background: 'white',
+        color: theme.brand,
+        border: `2px solid ${theme.brand}`,
+        margin: '0 2em',
+        '&:hover': {
+          color: '#600101',
+          border: `2px solid #600101`,
+          background: '#FFE9E7',
+        },
+      },
+    },
+    hideMobile: {
+      display: 'initial',
+      [theme.breakpoints.down('sm')]: {
+        display: 'none',
+      },
+    },
+  })
+);
+
+export default function MenuBar() {
+  const classes = useStyles();
+  const { isSharingScreen, toggleScreenShare } = useVideoContext();
+  const roomState = useRoomState();
+  const isReconnecting = roomState === 'reconnecting';
+  const { appState } = useAppState();
+
+  return (
```

Comment:
Much of this seems very similar to the MenuBar component. I wonder if we truly need a new MobileBottomMenuBar component. Can we use `<Hidden />` in MenuBar to get it to work on mobile?